### ISSUE TITLE
[React Component] if estimatedItemHeight provided it throws exception

### ISSUE THREAD

**Issue by felixmosh** [catamphetamine/virtual-scroller#7]
This layout
```jsx
<VirtualScroller
      items={list}
      estimatedItemHeight={350}
      itemComponent={AccommodationListItem}
/>
```

Throws: 
```
VirtualScroller.js:419 Uncaught TypeError: Cannot read property 'itemSpacing' of undefined
    at VirtualScroller.getItemSpacing (VirtualScroller.js:419)
    at VirtualScroller.getEstimatedItemsCount (VirtualScroller.js:425)
    at VirtualScroller.getEstimatedItemsCountOnScreen (VirtualScroller.js:434)
    at VirtualScroller.getLastShownItemIndex (VirtualScroller.js:442)
    at VirtualScroller.getInitialLayoutState (VirtualScroller.js:386)
    at VirtualScroller.getInitialState (VirtualScroller.js:367)
    at new VirtualScroller (VirtualScroller.js:346)
    at new ReactVirtualScroller (ReactVirtualScroller.js:111)
    at constructClassInstance (react-dom.development.js:14239)
    at updateClassComponent (react-dom.development.js:18350)
```

**Comment by catamphetamine** [comment]
You can provide an online demo.

**Comment by catamphetamine** [comment]
Actually, no need for a demo: I think I see how it errors.
Will fix today.

**Comment by felixmosh** [comment]
Wow! 
Thanx for the quick replay...🙏🏼

I've checked the reason for that, it access component state (with `getState`) which is `undefined`.

I can PR a fix for it.

**Comment by catamphetamine** [comment]
Ok, should be fixed in `virtual-scroller@1.0.25`.
Gone until evening.
